templates/template/Makefile

```diff
@@ -1,8 +1,11 @@
-CHARTS_BUILD_SCRIPT_VERSION := fdf0565
```

Comment:
This is tracked directly in a scripts/version file now. That way we won't automatically update the script version on a docs change.